examples/next-with-app-router/README.md

```diff
@@ -123,6 +123,38 @@ Create a file `src/app/globals.scss` and add the following import:
 @forward '@lmc-eu/spirit-web/scss';
 ```
 
+### PostCSS Configuration
+
+If you encounter the following warning:
+
+`autoprefixer: start value has mixed support, consider using flex-start instead`
+
+You can suppress this warning by customizing the PostCSS configuration. Create a `postcss.config.js` file in the root of your project with the following content:
+
+```js
+module.exports = {
+  plugins: {
+    autoprefixer: {
+      flexbox: 'no-2009',
+    },
+  },
+};
+```
+
+Alternatively, you can use a json configuration by creating a `postcss.config.json` file:
+
+```json
+{
+  "plugins": {
+    "autoprefixer": {
+      "flexbox": "no-2009"
+    }
+  }
+}
+```
+
+This configuration disables support for the outdated 2009 flexbox specification, which helps prevent compatibility warnings related to legacy flexbox syntax.
```

Comment:
I would rather move this paragraph above the examples. Could you please slightly rewrite it and insert it after the warning message? Thank you!